packages/react-material/src/picker/PickerTreeViewMaterial.jsx

```diff
@@ -70,22 +70,17 @@ let styles = theme => ({
     }
 });
 
-class PickerTreeViewMaterial extends React.Component {
+class PickerTreeViewMaterialCmp extends React.Component {
```

Comment:
In this component, the state and its usage in the bound functions has been removed but not replace. 
Either we replace it with something else, or we restore it